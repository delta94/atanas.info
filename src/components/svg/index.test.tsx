import * as React from 'react';
import { shallow } from 'enzyme';

import { Svg } from '.';

describe('Svg component', () => {
	it('Should render the Svg component', () => {
		const wrapper = shallow(<Svg src="test" className="test" />);

		expect(wrapper).toMatchSnapshot();
	});

	it('Should render the Svg component without an additional className', () => {
		const wrapper = shallow(<Svg src="test" />);

		expect(wrapper).toMatchSnapshot();
	});
});
