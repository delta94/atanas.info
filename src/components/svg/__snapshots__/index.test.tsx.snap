// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg component Should render the Svg component 1`] = `
<InlineSVG
  cacheRequests={true}
  className="c-svg-icon test"
  src="test"
  uniquifyIDs={false}
/>
`;

exports[`Svg component Should render the Svg component without an additional className 1`] = `
<InlineSVG
  cacheRequests={true}
  className="c-svg-icon "
  src="test"
  uniquifyIDs={false}
/>
`;
